src/transformers/modeling_tf_utils.py

```diff
@@ -808,20 +839,29 @@ def prune_heads(self, heads_to_prune):
         """
         raise NotImplementedError
 
-    def save_pretrained(self, save_directory):
+    def save_pretrained(self, save_directory, saved_model=True, version=1):
```

Comment:
The default should be `False` here IMO to not have backwards breaking changes without notifying the user. We can switch the flip in a future version. Also should this `saved_model` param maybe go in the config under a different name like `save_in_optimized_format` ? and we let the model default to the config?

Reply:
I don't mind having this parameter to false by default^^

Reply:
Yes, I would put it to `False` by default as well.